docs/en/user_guides/1_config.md

```diff
@@ -89,7 +89,7 @@ which is convenient to conduct various experiments.
   ```python
   # model settings
   model = dict(  # Config of the model
-      type='Recognizer2D',  # Type of the recognizer
+      type='Recognizer2D',  # Name of the recognizer
```

Comment:
What about 'Class name of the recognizer'?

Reply:
```suggestion
      type='Recognizer2D',  # Class name of the recognizer
```